### ISSUE TITLE
Parser failure with alias imports

### ISSUE THREAD

**Issue by 3nvi** [switchupcb/copygen#6]
Hey there! There is a failure when using aliased imports for certain model packages (see below). Removing the alias removes the shown error as well, albeit the conversion still doesn't work or me 😅 . 

I believe we make sure aliased imports work correctly, since many services have the same package name for their models

### YML

```yml
generated:
  setup: ./models/copygen.go
  output: ./models/mappers_gen.go

```

### Go

**THIS DOESN'T WORK**
```go
package models

import usermodels "github.com/..blahblah../users/models"

type Copygen interface {
	ModelsToDomain(usermodels.User) User
}
```

**THIS DOES**
```go
package models

import "github.com/..blahblah../users/models"

type Copygen interface {
	ModelsToDomain(models.User) User
}
```

## Output

No output

### Error

```
an error occurred while parsing the types of function "ModelsToDomain".
an error occurred while searching for the top-level Field "User" of package "usermodels".
the type declaration for the Field "User" of package "usermodels" could not be found in the AST.
Is the imported package up to date?
```

### Generation

```go
paste any generated code here.
```

## Environment

**Operating System**: OSX
**Copygen Version**: 0.2.1

**Comment by waltcow** [comment]
this issue still not resolved in latest version

**Comment by switchupcb** [comment]
@waltcow Can you post the setup with an issue a new issue? 

Alias imports works in the tests and in https://github.com/switchupcb/disgo/blob/main/wrapper/copygen/setup/setup.go.

Have you used the following command?
```
go install github.com/switchupcb/copygen@v0.3.7
```

**Comment by waltcow** [comment]
@switchupcb 
```
# Define where the code will be generated.
generated:
  setup: ./setup.go
  output: ../copygen.go

```

After recheck gererated file ,  `copygen.go` is generated but has ` Permission denied`  issue,

**Comment by waltcow** [comment]
![image](https://user-images.githubusercontent.com/3941174/172429921-f7315b0a-e701-4a5a-a84b-48b41bbd33e0.png)

**Comment by waltcow** [comment]
related to #26 

switch to main branch `go install github.com/switchupcb/copygen@main`,  problem solved

**Comment by switchupcb** [comment]
@waltcow Great! The permission denied error fix wasn't included in the `v0.3.7` release. I've created a `v0.3.8` release with an additional issue fix (not related to this issue). You can install it using `go install github.com/switchupcb/copygen@v0.3.8`.
